tests/test_clinvar.py

```diff
@@ -38,7 +38,7 @@ def test_observed_pubmed_refs(self):
         self.assertEqual(self.test_clinvar_record.observed_pubmed_refs, [11524702, 12145752])
 
     def test_clinical_significance(self):
-        self.assertEqual(self.test_clinvar_record.clinical_significance, "pathogenic")
+        self.assertEqual(self.test_clinvar_record.clinical_significance, "Pathogenic")
```

Comment:
The Open Targets evidence string json schema is case sensitive https://github.com/opentargets/json_schema/commit/70c643c62bd6e0e11b27922f63535ff30af7b6be#diff-01a2287f1c62aa9f2292520d43db54e6R16